checkClock/app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,274 @@
+          width: 25%;
+          padding: 0px 20px 0px 20px;
+          .employees, .offices{
+            margin-bottom: 25px;
+          }
+          .link-options{
+            width: 100%;
+            border-radius: 20px;
+            background-color: $color-primary;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            padding: 15px 0px;
+            p{
+              color: $font-color-secondary;
+              font-size: 14px;
+            }
+            a{
+              color: $font-color-primary;
+              text-decoration: none;
+              font-size: 19px;
+              letter-spacing: 2px;
+            }
+          }
+        }
+        .col-right{
+          width: 35%;
+          display: flex;
+          flex-direction: column;
+          justify-content: center;
+          align-items: center;
+          padding-top: 20px;
+          padding-bottom: 20px;
+          background-color: $color-primary;
+          border-radius: 20px;
+          box-shadow: 0 0 30px rgba(0, 0, 0, 0.25);
+          margin-right: 20px;
+          .date{
+            text-align: center;
+            font-weight: 400;
+            margin-bottom: 15px;
+            font-size: 1em;
+            letter-spacing: 2px;
+            color: $font-color-primary;
+          }
+          .hour{
+            font-size: 2em;
+            color: $font-color-primary;
+            #hour{
+              text-align: center;
+            }
+          }
+          .clock-container{
+            width: 150px;
+            height: 150px;
+            border: $test-boder;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            border-radius: 50%;
+            background-color: $color-secondary-light;
+            margin-bottom: 10px;
+            .clock-inside{
+              width: 120px;
+              height: 120px;
+              background-color: $color-secondary;
+              position: relative;
+              border-radius: inherit;
+              .minutes{
+                height: 35%;
+                width: 12px;
+                border-radius: none;
+                background-color: $color-secondary-light;
+                position: absolute;
+                border-radius: 10px;
+                top: 50%;
+                left: 50%;
+                transform: translateX(-50%) rotate(180deg); 
+                transform-origin: top;
+              }
+              .hours{
+                height: 27%;
+                width: 12px;
+                border-radius: 10px;
+                background-color: $color-primary;;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translateX(-50%) rotate(180deg); 
+                transform-origin: top;
+              }
+            }
+          }
+         }
+      }
+    }
+  }
+}
```

Comment:
the structure of this file can be better, if will be respect the lines break between each code block and if each block are sort ascending alphabetical, it would look better organized.

Reply:
Done, it's already sorted alphabetically